dashboard/alerts/alert.py

```diff
@@ -87,29 +119,41 @@ def saveAlert(self):
         """
         try:
             # Open config file to get path
-            config_file = open('/etc/capstone-ids/config.json', 'r')
-            config_data = json.load(config_file)
-            path = config_data["alert_collection_path"]
-
-            # Default path
-            if path == "" or path is None:
-                path = "/etc/capstone-ids/alert_collection.json"
+            try:
+                config_file = open('/etc/tinyHIPPO/config.json', 'r')
+                config_data = json.load(config_file)
+                path = config_data["alert_collection_path"]
+            except:
+                path = "/etc/tinyHIPPO/alert_collection.json"
 
             # Open alert collection file to read
-            alert_collection = open(path, 'r')
-            alert_data = json.load(alert_collection)
+            alert_collection = None
+            try:
+                alert_collection = open(path, 'r')
+                try:
+                    alert_data = json.load(alert_collection)
+                except json.decoder.JSONDecodeError:
+                    alert_data = json.loads("{\n \"alerts\": []\n}\n")
+            except FileNotFoundError:
+                alert_data = json.loads("{\n \"alerts\": []\n}\n")
 
             # Get the list of alerts, add the current object to the list
             alerts = alert_data["alerts"]
             alerts.append(self.jsonify())
 
             #Close/Open the file to RW properly (In case of earlier error, doesn't terminate the file's contents
-            alert_collection.close()
+            if alert_collection:
+                alert_collection.close()
```

Comment:
Using the "with open..." would prevent having to manually close the file descriptor here, IE
`with open(path, r) as alert_collection
.....
`

Reply:
"with open" would work, but also having the error handling mentioned above would lead to duplicated code when trying to pair it with the "with open" method of file access.

Reply:
See my above comment regarding handling those errors